doc/en/customize.rst

```diff
@@ -371,3 +371,9 @@ passed multiple times. The expected format is ``name=value``. For example::
 
 
 .. _`#3155`: https://github.com/pytest-dev/pytest/issues/3155
+
+.. confval:: rootdir
```

Comment:
I just realized that this is not really a configuration variable (e.g goes into a `pytest.ini` file) but a command line option.

I think it makes more sense to add a paragraph to the `Initialization: determining rootdir and inifile` section in this same file instead.